src/components/Menu.tsx

```diff
@@ -0,0 +1,41 @@
+import {Button, ButtonToolbar} from 'react-bootstrap';
+
+type ButtonType = {
+  title: string,
+  backgroundColor: string,
+  color: string
+}
```

Comment:
`;`

You should check all my comments for the all project. So, we shouldn't repeat ourselves.